pkgs/by-name/cu/cuelang/tests/001-all-good.nix

```diff
@@ -0,0 +1,17 @@
+{ lib
+, stdenvNoCC
+, cue
+}:
+
+stdenvNoCC.mkDerivation {
```

Comment:
I think `runCommand` would be preferable to `stdenvNoCC` with `buildCommand`?

Reply:
I never tried this. Let's see...

Reply:
Ugly, but it worked.